### PR TITLE
rygel: include the PipeWire GStreamer plugin

### PR DESCRIPTION
**Pull request by jwillikers**
The Rygel application can use the PipeWire GStreamer elements `pipewiresink` and `pipewiresrc`, but they are not currently available.
This PR just includes them.

The `package.nix` file was also formatted appropriately.

This is pretty much the same as #355207.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/ry/rygel/package.nix [hunk 1]
```diff
@@ -95,6 +96,12 @@ stdenv.mkDerivation (finalAttrs: {
     patchShebangs data/xml/process-xml.py
   '';
 
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "${pipewire}/lib/gstreamer-1.0"
```
Comment (jtojnar) [inline pkgs/by-name/ry/rygel/package.nix:107]: Why not just add it to `buildInputs` and have the hook pick it up automatically?

Comment (jwillikers) [reply pkgs/by-name/ry/rygel/package.nix:107]: Testing that, it's not added automatically to `GST_PLUGIN_SYSTEM_PATH_1_0`. I didn't see anything in the PipeWire package related to that either.

Comment (jwillikers) [reply pkgs/by-name/ry/rygel/package.nix:107]: Perhaps this is something that should be addressed in the PipeWire package and/or WrapGApps function?

Comment (bobby285271) [reply pkgs/by-name/ry/rygel/package.nix:107]: > Testing that, it's not added automatically to `GST_PLUGIN_SYSTEM_PATH_1_0`

Hmmm, I was looking the wrapper (`cat ./result/bin/rygel`) and don't expect this to happen? :thinking:

Comment (jwillikers) [reply pkgs/by-name/ry/rygel/package.nix]: @bobby285271 Thanks. My eyes totally glazed over the PipeWire plugin there in that output at first. I've got this:

```
    --prefix 'GST_PLUGIN_SYSTEM_PATH_1_0' ':' '/nix/store/mhp5jkwqgyz0s7klvpl23x3axgvn6msm-pipewire-1.2.6/lib/gstreamer-1.0:/nix/store/svr57k3p2bvw6a4kqsk6nyr6yrrsfsvw-gstreamer-1.24.7/lib/gstreamer-1.0:/nix/store/fya1nllqf03fm2fvxjl7m7x0gpnlhh8v-gst-plugins-base-1.24.7/lib/gstreamer-1.0:/nix/store/b28ks0qisdd7y7axvvzy3j5h1d24d4z7-gst-plugins-bad-1.24.7/lib/gstreamer-1.0:/nix/store/93dk53n3jjix4ym2mg4dp294qc31gv6q-gst-editing-services-1.24.7/lib/gstreamer-1.0:/nix/store/275gz6gnq3qdf2zkwa9kyql2jw3alszz-gst-plugins-good-1.24.7/lib/gstreamer-1.0:/nix/store/l75bsivp44jps2jqi1wwgirc25q8l179-gst-plugins-ugly-1.24.7/lib/gstreamer-1.0' 
```

Sorry for the confusion, I've updated the commit.

**Review by jtojnar — APPROVED** [review]

**Pull request merged**